builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -518,18 +517,11 @@ func (p *puller) Snapshot(ctx context.Context, g session.Group) (cache.Immutable
 
 	var nonLayers []digest.Digest
 
-	var (
-		schema1Converter *schema1.Converter
-		handlers         []c8dimages.Handler
-	)
+	var handlers []c8dimages.Handler
 	if p.desc.MediaType == c8dimages.MediaTypeDockerSchema1Manifest {
-		schema1Converter, err = schema1.NewConverter(p.is.ContentStore, fetcher)
-		if err != nil {
-			stopProgress()
-			return nil, err
-		}
-		handlers = append(handlers, schema1Converter)
-
+		stopProgress()
+		// similar to [github.com/docker/docker/distribution/DeprecatedSchema1ImageError]
+		return nil, cerrdefs.ErrInvalidArgument.WithMessage(errors.New("support for Docker Image Format v1 and Docker Image manifest version 2, schema 1 has been removed. More information at https://docs.docker.com/go/deprecated-image-specs/"))
```

Comment:
```suggestion
		return nil, cerrdefs.ErrInvalidArgument.WithMessage(errors.New("support for Docker Image Format v1 and Docker Image manifest version 2, schema 1 has been removed in Docker Engine v28.2. More information at https://docs.docker.com/go/deprecated-image-specs/"))
```

Reply:
Ah, yes, can change.

Looks like I messed this up as well as I changed this last-minute 😂 

```
builder/builder-next/adapters/containerimage/pull.go:524:55: cannot use errors.New("support for Docker Image Format v1 and Docker Image manifest version 2, schema 1 has been removed. More information at https://docs.docker.com/go/deprecated-image-specs/") (value of interface type error) as string value in argument to cerrdefs.ErrInvalidArgument.WithMessage
```